client/components/forms/form-phone-media-input/index.jsx

```diff
@@ -17,10 +17,17 @@ export default class extends React.Component {
 		isError: false,
 	};
 
+	focus = () => {
+		this.phoneInput.numberInput.focus();
+	}
+
+	setPhoneInput = ( ref ) => this.phoneInput = ref;
```

Comment:
OK, I'm nitpicking here but... I think we should be consistent in how we define methods. So `this.phoneInput = ref` should be in `{...}` block on a new line or otherwise we should fix the `focus` method to be one liner too

Reply:
No...good point. You are correct!